types/vinyl-fs/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "dependencies": {
+      "rimraf": "^4.0.0"
```

Comment:
this is probably not right,
what's the version of rimraf used by native vinyl-fs module? The version 4.0 introduce modules (ESM/CJS)

Reply:
You are right!